.github/workflows/CI.yml

```diff
@@ -24,21 +24,26 @@ jobs:
     - name: Install GitVersion
       uses: gittools/actions/gitversion/setup@v0.9.7
       with:
-        versionSpec: '5.x'
+        versionSpec: 5.x
+
     - name: Determine Version
       uses: gittools/actions/gitversion/execute@v0.9.7
       id: gitversion # step id used as reference for output values
+
     - name: Display GitVersion outputs
       run: |
-        echo "MajorMinorPatch: ${{ steps.gitversion.outputs.majorMinorPatch }}"
-        echo "NuGetVersionV2: ${{ steps.gitversion.outputs.nuGetVersionV2 }}"
+        echo "semVer: ${{ steps.gitversion.outputs.semVer }}"
         echo "CommitsSinceVersionSource: ${{ steps.gitversion.outputs.CommitsSinceVersionSource }}"
     
-    #Build and test code, then upload code coverage to coverall.io
-    #- name: Build with dotnet
-    #  run: dotnet build src/AzurePipelinesToGitHubActionsConverter.Tests/AzurePipelinesToGitHubActionsConverter.Tests.csproj --configuration Release /p:AssemblyVersion=${{ steps.gitversion.outputs.majorMinorPatch }} /p:Version=${{ steps.gitversion.outputs.majorMinorPatch }}
-    - name: Test with dotnet
-      run: dotnet test src/AzurePipelinesToGitHubActionsConverter.Tests/AzurePipelinesToGitHubActionsConverter.Tests.csproj --configuration Release /p:CollectCoverage=true /p:CoverletOutput=TestResults/ /p:CoverletOutputFormat=lcov
+    - name: dotnet restore
+      run: dotnet restore --verbosity minimal --configfile nuget.config
+
+    - name: dotnet build
```

Comment:
Assuming we need the build to debug - why is it release? Also, why not combine the restore with the build?

Reply:
Not sure of the reference to release in this comment - but when users consume your package they will generally want to implement a Release version for (slight) performance increase and smaller dlls in their published packages (the inclusion of SourceLink+Symbols will give them the ability to debug).
Not combined into one CI step because it gives a (simple and understandable) separation of dotnet workflow. - helps teach developers that `dotnet <anything>` generally does an implicit restore (which can slow things down and duplicate effort).

Reply:
Clarifying my question: do we *need* these steps for the source nuget package? If not, let's remove them and simplify the yaml file. This isn't a complex series of steps.

Reply:
No you don't need them, but given how you educating via your blog...
See section 'Implicit Builds' on https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build

Reply:
Let's keep this part optimized, and drop the restore step. I'll clone down the branch and take care of it.